README.md

```diff
@@ -57,3 +69,32 @@ The composition tool fetches and installs dependencies for a package by recursiv
 5. Repeat steps 1-4 for each package and its dependencies.
 
 During this process, the composition tool keeps track of the dependencies for each package.
+
+### Plasma-compose commands
+it's possible to manipulate plasma-compose.yaml file using commands:
+- plasmactl compose:add
+- plasmactl compose:update
+- plasmactl compose:delete
+
+For `compose:add` and `compose:update` there are 2 ways to submit data. With or without flags.
+Passing `--package` and `--url` to add command will automatically update plasma-compose file.
+For update command only `--package` required to update from CLI.
+
+For `compose:delete` it's possible to pass list of packaged to delete.
+
+In other cases, user will be prompted to CLI form to fill necessary data of packages.
+
+Example of usage
+
+
+```
+launchr compose:add --url some-url --type http
+launchr compose:add --package package-name --url some-url --ref v1.0.0
+launchr compose:update --package package-name --url some-url --ref v1.0.0
+
```

Comment:
Looks like there is an issue with strategies when passed as option:

For instnance:
```
./plasmactl compose:add --package package-name --url some-url --ref v1.0.0 --strategy overwrite-local-file,remove-extra-local-files --strategy-path "path1|path2,path3|path4"                                          
```
renders as:
```
+    - name: package-name
+      source:
+        type: git
+        url: some-url
+        ref: v1.0.0
+        strategy:
+            - name: overwrite-local-file
+              path:
+                - path2
+                - path2
+            - name: remove-extra-local-files
+              path:
+                - path3
+                - path3
```
You can see that `path1` and `path4` passed as option values are not written to file